src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/layout.tsx

```diff
@@ -27,7 +28,7 @@ export default async function Layout({ children }: { children: ReactNode }) {
   const headersList = await headers();
   const countryCode = getCountryCode(headersList);
 
-  if (!session) {
+  if (!session || !session?.user?.subscriber?.id) {
```

Comment:
Curious if you've seen this in practice? It'd be nice to have a function to check this so we can ensure it's done consistently (similar to the `isAdmin` one)

Reply:
I think it /shouldn't/ happen that there's a `session` with no `subscriber.id`, so if we had a function I'd want to do a `logger.warn` too so we can see how often that is happening.